packages/sn-filter-pane/src/ext/conversion/index.ts

```diff
@@ -0,0 +1,44 @@
+import { ExportFormat, PropTree } from './types';
+
+export function createImportProperties(exportFormat: ExportFormat, initialProperties: PropTree) {
+  // TODO: add properties for unquarantine
+
+  const propTree: PropTree = {
+    qChildren: [
+      {
+        qProperty: {
+          qInfo: { qType: 'listbox' },
+        },
+      },
+    ],
+    qProperty: {
+      qInfo: {
+        qType: 'filterpane',
+      },
+    },
+  };
+  return propTree;
+}
+
+export function exportProperties(expFormat: ExportFormat) {
+  expFormat.data = [
```

Comment:
To avoid the warning seen here, create a shallow copy first (although that's not how the old code does it).
```
const out = { ...expFormat };
```